frontend/src/theme/components/button.ts

```diff
@@ -0,0 +1,90 @@
+const Button = {
```

Comment:
Might be helpful to document this (unless it's already somewhere I'm missing)

(eg. gif that shows what each button looks like + interaction)

Reply:
That's kind of documented here, perhaps I could take screenshots and put it in Notion? https://www.figma.com/file/zh6zk2LJo0RodIHBVycylC/RHS-Design-System?node-id=209%3A13640

Reply:
I'll create a Notion page outlining the possible variants.